crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -14,23 +14,22 @@
 # limitations under the License.
 #
 
-class Api::StoragesController < ApiController
+class Api::CephController < ApiController
   before_action :set_ceph
 
-  api :GET, "/api/storages", "List all Ceph storages"
+  api :GET, "/api/ceph", "Show the Ceph object"
```

Comment:
Hm, I do not like the "object", it looks like something inside ceph storage. What about "configuration"? (Not much better...)